code/drasil-example/Drasil/DblPendulum/Concepts.hs

```diff
@@ -4,7 +4,8 @@ import Language.Drasil
 
 import Data.Drasil.Domains (physics)
 
-
+pendulum :: CI
+pendulum = commonIdeaWithDict "pendulumTitle" (pn "Pendulum") "Pendulum" [physics]
```

Comment:
As this is now the concept a pendulum, the id "pendulumTitle" is not really apt anymore.  Can you make it just "pendulum"?